config/config_test.go

```diff
@@ -49,6 +49,14 @@ func TestDoesRequestSatisfyTermination(t *testing.T) {
 	r = &http.Request{Header: http.Header{"X-Forwarded-Proto": []string{"http"}}, URL: new(url.URL)}
 	assert.Error(t, c.DoesRequestSatisfyTermination(r))
 
+	r = &http.Request{
+		Header: http.Header{
```

Comment:
This test could also fail because no remoteaddress is given (thus SplitHostPort returns an error)

Reply:
Sorry, this was meaningless test...